variables.tf

```diff
@@ -4,6 +4,10 @@ variable "repo_name" {
 variable "team_name" {
 }
 
+variable "enable_scanning_on_push" {
+  default = true
```

Comment:
Can we add here?

```
type = bool
```

It will be helpful when we implement auto-documentation process (read this file and update the README.md). More info: https://github.com/antonbabenko/pre-commit-terraform